src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf

```diff
@@ -249,22 +249,22 @@
       </trans-unit>
       <trans-unit id="ERR_DefaultLiteralNoTargetType">
         <source>There is no target type for the default literal.</source>
-        <target state="translated">默认文本没有目标类型。</target>
+        <target state="translated">default 字面量缺少目标类型。</target>
```

Comment:
@AlekseyTs Looks like this is one of the place that would have benefit from a resource comment to prevent localization for keywords. I'd love if the team could discuss using these comments.